pkg/models/trb_request.go

```diff
@@ -3,10 +3,11 @@ package models
 // TRBRequest represents a TRB request object
 type TRBRequest struct {
 	baseStruct
-	Name     string           `json:"name" db:"name"`
-	Archived bool             `json:"archived" db:"archived"`
-	Type     TRBRequestType   `json:"type" db:"type"`     //TODO should this be a type?
-	Status   TRBRequestStatus `json:"status" db:"status"` //TODO should this be a type?
+	Name      string                `json:"name" db:"name"`
+	Archived  bool                  `json:"archived" db:"archived"`
+	Type      TRBRequestType        `json:"type" db:"type"`     //TODO should this be a type?
+	Status    TRBRequestStatus      `json:"status" db:"status"` //TODO should this be a type?
+	Attendees []*TRBRequestAttendee `json:"attendees"`
```

Comment:
@jstrang-oddball , I don't think Attendees should actually be in the model here, instead it would just be in the GQL type. 

The benefit for that is you don't need to query it unless asked for by GQL. This is how we handled it similarly in MINT. [model_plan struct here](https://github.com/CMSgov/mint-app/blob/main/pkg/models/model_plan.go#L8).  

The [plan collaborator struct is here](https://github.com/CMSgov/mint-app/blob/main/pkg/models/plan_collaborator.go).


GQL Gen sees that [collaborator](https://github.com/CMSgov/mint-app/blob/main/pkg/graph/schema.graphql#L42) from the schema definition of model plan isn't implemented, so it makes a resolver in [schema.resolvers.go](https://github.com/CMSgov/mint-app/blob/main/pkg/graph/schema.resolvers.go#L54-L61).

With this logic, you don't need to add logic to the parent resolver to always return the attendees, GQL will handle it only when requested. It still results in multiple db queries, but only when requested.